### PR TITLE
META-993 - MetadataFIBO does not load all of the FIBO content

### PR DESCRIPTION
**Pull request by ElisaKendall**
Signed-off-by: Elisa Kendall ekendall@thematix.com

## Description

Revised metadata files so that the provisional ones were consistent with what we have in release, and to ensure that none are missing if one imports MetadataFIBO.  MetadataFIBO can be used together with AboutFIBODev to drive the FIBO viewer as a consequence.

Fixes: #993 

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](../CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](../ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

File: MD/TemporalCore/MetadataMDTemporalCore.rdf [hunk 1]
```diff
@@ -22,14 +22,14 @@
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/MD/TemporalCore/MetadataMDTemporalCore/">
 		<rdfs:label>Metadata for the EDMC-FIBO Market Data (MD) TemporalCore Module</rdfs:label>
 		<dct:abstract>This is the metadata ontology used to describe the MD Temporal Core Module.</dct:abstract>
```
Comment (rivettp) [inline MD/TemporalCore/MetadataMDTemporalCore.rdf:24]: Not duplicating the abstract from the Module?

**Comment by trypuz** [comment]
Indeed MetadataFIBO and AboutFIBODev make FIBO Viewer work correctly. We should make sure that all relevant meta-ontology files are imported and all of them provide up-to-date info about FIBO ontologies.

**Comment by ElisaKendall** [comment]
> There are extensive changes that have nothing to do with the purported issue of loading all the FIBO content.
> In fact the bulk of the changes consist of duplicating values for dct:abstracts which a) is a change snuck in with no description or justification or even commit comment and b) is sure to result in maintenance issues since I'm sure one will be updated without the other. The Don't Repeat Yourself (DRY) principle should apply! In fact in at least one case (CAECorporateEvents) this PR makes them inconsistent.
> 
> Even worse, the PR, if I understand the intent correctly, does not for many modules do what I'd expect to address the issue, and import the ontologies which are listed under hasPart for the Modules (I commented on a couple of specific cases but not all).
> 
> The addition or rdfs:seeAlso triples with value of spec.edmcouncil.org/fibo/ while not illegal is bad practice IMO, since as W3C says https://www.w3.org/wiki/UsingSeeAlso "It is reasonable to expect many (probably even "most") rdfs:seeAlso properties to reference to RDF/XML documents. ". Given that we do have a proper RDF resource describing FIBO I strongly feel we should reference that i.e. the MetadataFIBO resource.

Just to follow up, since I responded to this in the issue, Robert and I agreed that the MetadataFIBO SHOULD NOT import the FIBO ontologies, and should be self-contained, but some of the files had horrible metadata (that said, essentially, this is metadata), all in provisional, and at least one of them imported the wrong thing - it imported an ontology not the metadata ontology that it was supposed to reference.  This resolution corrects precisely and only those issues, all in provisional metadata files that can be easily fixed again downstream as the ontologies themselves are revised.  

Note too that I anticipate a number of these provisional modules, though not all, will be deleted as we merge and integrate the ontologies.

File: CAE/CorporateEvents/MetadataCAECorporateEvents.rdf [hunk 2]
```diff
@@ -20,25 +20,27 @@
 	xmlns:xsd="http://www.w3.org/2001/XMLSchema#">
 	
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/CAE/CorporateEvents/MetadataCAECorporateEvents/">
-		<rdfs:label>Metadata for the EDMC-FIBO Corporate Actions and Events (CAE) CorporateEvents Module</rdfs:label>
-		<dct:abstract>This is the metadata ontology used to describe the CAE Corporate Events Module.</dct:abstract>
-		<dct:issued rdf:datatype="&xsd;dateTime">2018-06-30T18:00:00</dct:issued>
-		<dct:license rdf:resource="&sm;MITLicense"/>
+		<rdfs:label>Metadata for the EDMC-FIBO Corporate Actions and Events (CAE) Corporate Events Module</rdfs:label>
+		<dct:abstract>The Corporate Actions and Events (CAE) Corporate Events Module is the primary module for the CAE domain. It covers events and actions that may occur during the life of a security, ranging from announcements regarding stock offerings, splits, dividends and so forth, to credit events that are relevant to investors and regulators alike.</dct:abstract>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/CorporateEvents/MetadataCAECorporateEvents.rdf:24]: fixed

File: CAE/CorporateEvents/MetadataCAECorporateEvents.rdf [hunk 3]
```diff
@@ -20,25 +20,27 @@
...
 		<rdf:type rdf:resource="&sm;Module"/>
-		<rdfs:label>Corporate Events</rdfs:label>
+		<rdfs:label>Corporate Events Module</rdfs:label>
 		<dct:abstract>This module contains ontologies of general Corporate Events concepts including events, messages, activities and responses required from security holders.</dct:abstract>
 		<dct:hasPart rdf:resource="https://spec.edmcouncil.org/fibo/ontology/CAE/CorporateEvents/CorporateActionsEvents/"/>
-		<dct:title>FIBO Corporate Actions and Events - Corporate Events Module</dct:title>
-		<sm:moduleAbbreviation>fibo-cae-ce</sm:moduleAbbreviation>
+		<dct:license rdf:datatype="&xsd;anyURI">http://opensource.org/licenses/MIT</dct:license>
+		<sm:copyright>Copyright (c) 2013-2020 EDM Council, Inc.</sm:copyright>
+		<sm:moduleAbbreviation>FIBO-CAE-CE</sm:moduleAbbreviation>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/CorporateEvents/MetadataCAECorporateEvents.rdf:42]: I think it's ad hoc - some are upper case, some are lower case.  fixed.

File: CAE/MetadataCAE.rdf [hunk 4]
```diff
@@ -22,37 +22,28 @@
 	xmlns:xsd="http://www.w3.org/2001/XMLSchema#">
 	
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/CAE/MetadataCAE/">
-		<rdfs:label>Metadata for the EDMC-FIBO Corporate Actions and Events (CAE) Domain</rdfs:label>
-		<dct:abstract>The metadata for CAE ontology describes the CAE domain.</dct:abstract>
+		<rdfs:label>Metadata about the EDMC-FIBO Corporate Actions and Events (CAE) Domain</rdfs:label>
+		<dct:abstract>The Corporate Actions and Events (CAE) domain covers events and actions that may occur during the life of a security, ranging from announcements regarding stock offerings, splits, dividends and so forth, to credit events that are relevant to investors and regulators alike. Corporate actions include actions that require some action on the part of the holder, and in these and some other cases there are process descriptions for the flow of activities involved.</dct:abstract>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/MetadataCAE.rdf:26]: That's Mike's definition - fixed by using a slightly revised abstract that was in the named individual.

File: BP/Process/MetadataBPProcess.rdf [hunk 5]
```diff
@@ -21,24 +21,26 @@
 	
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/BP/Process/MetadataBPProcess/">
 		<rdfs:label>Metadata for the EDMC-FIBO Business Process (BP) Process Module</rdfs:label>
-		<dct:abstract>This is the metadata ontology used to describe the BP Process Module.</dct:abstract>
-		<dct:issued rdf:datatype="&xsd;dateTime">2018-06-30T18:00:00</dct:issued>
-		<dct:license rdf:resource="&sm;MITLicense"/>
+		<dct:abstract>This module contains ontologies of Process concepts including concepts common to a range of business processes, along with basic financial context concepts.</dct:abstract>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply BP/Process/MetadataBPProcess.rdf:24]: This was MB's original definition - I can change it, or delete the module altogether, but that would require addressing dependencies which I think should be done in a separate issue.

File: CAE/CorporateEvents/MetadataCAECorporateEvents.rdf (hunk 2 above)
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/CorporateEvents/MetadataCAECorporateEvents.rdf:24]: fixed.

File: LOAN/MetadataLOAN.rdf [hunk 6]
```diff
@@ -35,19 +35,18 @@
 		<sm:copyright>Copyright (c) 2014-2020 EDM Council, Inc.</sm:copyright>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply LOAN/MetadataLOAN.rdf:35]: fixed

File: MD/CIVTemporal/MetadataMDCIVTemporal.rdf [hunk 7]
```diff
@@ -21,24 +21,27 @@
 	
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/MD/CIVTemporal/MetadataMDCIVTemporal/">
 		<rdfs:label>Metadata for the EDMC-FIBO Market Data (MD) CIVTemporal Module</rdfs:label>
-		<dct:abstract>This is the metadata ontology used to describe the MD CIV Temporal Module.</dct:abstract>
-		<dct:issued rdf:datatype="&xsd;dateTime">2018-06-30T18:00:00</dct:issued>
-		<dct:license rdf:resource="&sm;MITLicense"/>
+		<dct:abstract>This module contains ontologies of temporally variable concepts relating to funds and other collective investment vehicles. These are concepts for variables that have past, presented and projected future values.</dct:abstract>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply MD/CIVTemporal/MetadataMDCIVTemporal.rdf:24]: The entire module needs to be eliminated and the relevant content merged into the Funds ontology.  It's possible that the funds ontology itself needs to be refactored into more than one ontology, though.

I've attempted to fix this a bit - this was MB's original abstract that I didn't touch initially.

File: MD/DebtTemporal/MetadataMDDebtTemporal.rdf [hunk 8]
```diff
@@ -21,15 +21,15 @@
 	
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/MD/DebtTemporal/MetadataMDDebtTemporal/">
 		<rdfs:label>Metadata for the EDMC-FIBO Market Data (MD) DebtTemporal Module</rdfs:label>
-		<dct:abstract>This is the metadata ontology used to describe the MD Debt Temporal Module.</dct:abstract>
-		<dct:issued rdf:datatype="&xsd;dateTime">2018-06-30T18:00:00</dct:issued>
-		<dct:license rdf:resource="&sm;MITLicense"/>
+		<dct:abstract>This module contains ontologies of temporally variable concepts relating to debt instruments. These are concepts for variables that have past, presented and projected future values, such as pricing, yields and analytics.</dct:abstract>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply MD/DebtTemporal/MetadataMDDebtTemporal.rdf:24]: fixed

File: MD/DerivativesTemporal/MetadataMDDerivativesTemporal.rdf [hunk 9]
```diff
@@ -21,15 +21,15 @@
 	
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/MD/DerivativesTemporal/MetadataMDDerivativesTemporal/">
 		<rdfs:label>Metadata for the EDMC-FIBO Market Data (MD) DerivativesTemporal Module</rdfs:label>
-		<dct:abstract>This is the metadata ontology used to describe the MD Derivatives Temporal Module.</dct:abstract>
-		<dct:issued rdf:datatype="&xsd;dateTime">2018-06-30T18:00:00</dct:issued>
-		<dct:license rdf:resource="&sm;MITLicense"/>
+		<dct:abstract>This module contains ontologies of temporally variable concepts relating to derivatives instruments. These are concepts for variables that have past, presented and projected future values, such as the various derivatives related greeks and other analytics.</dct:abstract>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply MD/DerivativesTemporal/MetadataMDDerivativesTemporal.rdf:24]: fixed

File: MD/TemporalCore/MetadataMDTemporalCore.rdf (hunk 1 above)
Comment (ElisaKendall) [reply MD/TemporalCore/MetadataMDTemporalCore.rdf:24]: fixed.

**Pull request merged**